pyomo/contrib/incidence_analysis/config.py

```diff
@@ -54,7 +65,90 @@ class IncidenceMethod(enum.Enum):
 )
 
 
-IncidenceConfig = ConfigDict()
+class _ReconstructVisitor:
+    pass
```

Comment:
At some point I needed a singleton to trigger re-construction of `IncidenceConfig`'s visitor. I'm not sure if this is still necessary, and will re-visit.